internal/serverinstall/docker.go

```diff
@@ -95,6 +96,28 @@ func (i *DockerInstaller) Install(
 		s.Update("Detected existing Waypoint server.")
 		s.Status(terminal.StatusWarn)
 		s.Done()
+
+		for _, container := range containers {
```

Comment:
I think it'd be worthwhile to add a comment above this block to explain what its for.